test/JDBC/expected/BABEL-5119-vu-verify.out

```diff
@@ -0,0 +1,186 @@
+-- tsql
+CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
+CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
```

Comment:
why login creation in this file?

Reply:
Currently in babelfish, during upgrades logins are dumped without passwords so it is required to reset their passwords on the target server otherwise authentication will fail for those logins. So I created the logins in the verify itself.